nixos/modules/system/etc/etc.nix

```
@@ -6,22 +6,19 @@ with lib;
 
 let
 
-  etc' = filter (f: f.enable) (attrValues config.environment.etc);
+  etc' = lib.filter (f: f.enable) (lib.attrValues config.environment.etc);
```

The `lib.` is an unnecessary addition, since this file uses `with lib;` at the very top already (line 5).

```suggestion
  etc' = filter (f: f.enable) (attrValues config.environment.etc);
```

---

changed